src/main/resources/db/changelog/changes/create-book-table.yml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_book_table
+      author: bodya4243
+      changes:
+        - createTable:
+            tableName: book
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
primary key is not null by default in MySQL, so we can remove that or leave it if you want to explicitly declare it